src/main/kotlin/com/constructor/database/TableModel.kt

```diff
@@ -0,0 +1,85 @@
+package com.constructor.database
+
+import com.constructor.db_conn
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.util.UUID
+
+object TableModel : Table("test_table") {
+    val id = uuid("id")// must be uuid
+    val strfield = varchar("strfield", 30)
+    val intField = integer("intfield")
+    val boolField = bool("boolfield")
+    override val primaryKey = PrimaryKey(id, name = "PK_test_table_id")
+
+    fun RouteToDTO (rr : ResultRow) : TableDTO
+    {
+        return TableDTO(id = rr[TableModel.id],
+            stringField = rr[TableModel.strfield],
+            intfield = rr[TableModel.intField],
+            boolfield = rr[TableModel.boolField])
+    }
+
+    // crud
+    fun readAll() : MutableList<TableDTO>
+    {
+        val temp : MutableList<TableDTO> = mutableListOf()
+        transaction (db_conn){
+            val query = TableModel.selectAll()
+            val result = query.toList()
+            for(element in result)
+            {
+                temp.add(RouteToDTO(element))
+            }
+        }
+        return temp
+    }
+
+    fun Create(js : MutableList<TableDTO>)
```

Comment:
Название функции и переменных не по код стайлу
```suggestion
    fun сreate(tableDtoList : MutableList<TableDTO>)
```

Reply:
Да, еще вчера правил локально. Сейчас еще раз прочекал весь свой код, поправил где надо.